homework/style.css

```diff
@@ -1,3 +1,117 @@
 .alert-error {
     color: red;
+    text-align: center;
+    font-size: 4vw;
+    box-shadow:0 0 10px rgb(217, 255, 0); 
+    border-radius: 30px;
+    width: 80%;
+    margin: 0 auto;
+    padding: 20px;
+    margin-top: 20px;
+}
+* {
+    margin: 0;
+    padding: 0;
+}
+ body {
+    /* max-width: 1000px; */
+    min-width: 319px;
+    margin: 50px auto;
+    background-color: rgb(0, 0, 0);
+    animation: appear-main 5s;
+}
+@keyframes appear-main {
+  from {opacity: 0;}
+  to {opacity: 1;}
+}
+.HYF_Repositories{
+  text-align: center;
+  color: rgb(217, 255, 0);
+  text-shadow:  0 0 10px rgb(217, 255, 0);
+  box-shadow: 0 0 20px rgb(255, 0, 0);
+  padding: 20px;
+  border-radius: 20px;
+  margin: auto;
+  width: 80%;
+}
+select{
+border-radius: 10px;
+font-size: 20px;
+font-family: 'Times New Roman', Times, serif;
+margin-top: 10px;
+box-shadow: 0 0 20px rgb(255, 0, 0);
+} 
+.information_box{
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: center;
+  margin-top: 50px;
+}
+ ul { 
+   width:50%;
+   list-style: none;
+   font-family: 'Times New Roman', Times, serif;
+   font-size: 20px;
+   color:rgb(217, 255, 0);
+   text-shadow: 0 0 3px rgb(217, 255, 0);
+   box-shadow: 0 0 20px rgb(255, 0, 0);
+   margin-left: 20px;
+   padding: 20px;
+   text-align: center;
+   border-radius: 30px;
+  }
+  .right_ul{
+width: 25%;
+  }
+img {
+    height: 70px;
+    border-radius: 10px;
+    box-shadow: 0 0 20px rgb(217, 255, 0);
+}
+li {
+   padding: 25px;
+   border-radius: 20px;
+   box-shadow: 0 0 10px rgb(217, 255, 0);
+   margin-top: 10px; 
+   margin-left: 20px;
+   margin-right: 20px;
```

Comment:
Do a search for css shorthands, these three lines could be written in 1: `margin: 10px 20px 0;`